inc/class-statify-frontend.php

```diff
@@ -318,4 +318,34 @@ public static function wp_footer() {
 			)
 		);
 	}
+	
+	/**
+	* Add amp-analytics, see https://github.com/Automattic/amp-wp/wiki/Analytics
+	*/
+	public static function amp_post_template_analytics( $analytics ) {
+		
+		if ( ! is_array( $analytics ) ) {
+			$analytics = array();
+		}
+
+		$analytics['statify'] = array(
+			'type' => 'statify',
+			'attributes' => array(),
+			'config_data' => array(
+				'requests' => array(
+					'event' => get_site_url() . '/?statify_referrer=${documentReferrer}&statify_target=${canonicalPath}amp/'
```

Comment:
Maybe we should sanitize this url, like `esc_url()`.

Reply:
We probably shouldn't. The placeholders `${documentReferrer}` and `${canonicalPath}` are AMP builtins and replaced on client side and all other parts are either WP builtins or hardcoded.

Generated JSON by the AMP plugin:
```json
{
  "requests": {
    "event": "https:\/\/example.com\/?statify_referrer=${documentReferrer}&statify_target=${canonicalPath}amp\/"},
    "triggers": { "trackPageview": { "on": "visible", "request": "event", "vars": { "eventId": "pageview" } } 
  }
}
```

Escaping the braces here would yield something like `.../?statify_referrer=$documentReferrer&#038;statify_target=$canonicalPathamp/` which no longer provides the desired result.